data-roadmap/data_roadmap/data_set_description_field_descriptions.yaml

```diff
@@ -1,39 +1,23 @@
 # There is no method for adding field descriptions to `yamale` schemas.
 # Therefore, we've created a dictionary here of fields and their descriptions.
-name: A short name of the data set.
```

Comment:
We've ended up updating the descriptions directly in the Google spreadsheet. We should update this yaml file with those descriptions.